coveo-pyproject/README.md

```diff
@@ -1,240 +1,196 @@
-
-### build some-project
-This command outputs the artifacts of your project into a folder, so that it can be distributed based on its `poetry.lock`
-metadata.
-
-This is different from publishing to a `pypi` server, where distribution is based on the _constraints_ you placed
-in `pyproject.toml`.
-
-A nice bonus is that the folder does not require poetry or coveo-pyproject to be installed, so it helps keeping production
-systems as small and specific as they should be.
-
-To install from the artifacts folder: 
-`pip install --no-index --find-links /path/to/.wheels <package-name>`
-
-Without a project name specified, all projects will be built.
-
-Scroll down to the poetry gotchas for a more detailed use case and why this command exists!
 
+# Features
 
-### check-outdated
-Ensure that all files in the repo are up to date and return an error if something's outdated.
+## poetry on steroids / offline distribution
+- Orchestrates several isolated poetry projects in one repo
+- Supports unpublished, local-only libraries and projects
+- Generates offline installation package from lock files
 
-A file is out of date if:
-- the `pyproject.toml` changed but not the `poetry.lock`
-- a `pydev` project's dev requirements changed and `pull-dev-requirements` was not called.
+## developer tools
+- Act on python projects in a git repo by name (path is irrelevant)
+- Batch support for common operations such as poetry lock and poetry install
+- Specialized development environment support (called pydev)
 
+## ci tools
+- Builtin, config-free pytest and mypy checks
+- Able to resolve some checks automatically (e.g.: lock file is outdated? call poetry lock!)
+- JUnit report generation
+- Github action
 
-### fix-outdated
-Like `pyproject check-outdated`, but also fixes anything that is outdated.
 
+# Installation
 
-### bump some-project
-Bumps the version of all dependencies of _some-project_.
-
-Without a project specified, all lock files of the repository will be updated. This doesn't update constraints, rather, it
-bumps everything to the latest version that match your constraints.
-
-
-### pull-dev-requirements
-`poetry install` will not install the dev-requirements of dependencies...
-
-If you have a `pyproject.toml` that is aimed at unifying all python projects of a 
-repository inside a single environment, run this script to pull the dev requirements
-of local dependencies inside that file.
-
-To mark such a development-only environment, add this to its `pyproject.toml` file:
+It is recommended to install using `pipx` in order to isolate this into a nice little space:
 
 ```
-[tool.coveo.pyproject]
-pydev = true
+pip install pipx --user
+pipx install coveo-pyproject
 ```
 
-Note: pydev projects cannot (and should not) be built or published.
+If you don't use pipx, make sure to isolate the installation into a virtual environment, else it may interfere with an existing poetry installation.
```

Comment:
```suggestion
If you don't use pipx, make sure to isolate the installation into a virtual environment, otherwise it may interfere with an existing poetry installation.
```